src/OrchardCore.Modules/OrchardCore.Lucene/Handler/LuceneIndexingContentHandler.cs

```diff
@@ -82,7 +81,8 @@ private static async Task IndexingAsync(ShellScope scope, IEnumerable<ContentCon
 
                 foreach (var indexSettings in await luceneIndexSettingsService.GetSettingsAsync())
                 {
-                    var ignoreIndexedCulture = indexSettings.Culture == "any" ? false : context.ContentItem.Content?.LocalizationPart?.Culture != indexSettings.Culture;
+                    var culture = await orchardHelper.GetContentCultureAsync(context.ContentItem);
```

Comment:
Don't use the `IOrchardHelper` here. Use the `IContentManager.PopulateAspect`

The helper methods are for the ui/users of OC, not internal code.